10up/brightcove-video-connect
# Add settings sidebar to the block

### Description of the Change

This change adds the settings sidebar to the block.

![settings bar](https://user-images.githubusercontent.com/6010232/133997250-b3fa3b2c-5199-45d9-87e1-bc5f281e4aad.png)


### Alternate Designs



### Benefits

The use can the settings without open the modal.

### Possible Drawbacks



### Verification Process

1. Add a video/playlist using the Brightcove block
2. Select the block and change the settings on sidebar
3. Visit the post and check if the changes are reflected on the frontend

### Checklist:






### Applicable Issues



### Changelog Entry

Added settings sidebar to the block
